File: app/models/question.py
from django.db import models
from app.models.week import Week

class Question(models.Model):
    def __str__(self):
        return (self.question_text[:60] + '..') if len(self.question_text) > 60 else self.question_text

    question_id = models.BigAutoField(primary_key=True)
    question_text = models.TextField()
    week = models.ForeignKey(Week, on_delete=models.CASCADE, null=True)

File: express/models/Question.js
const SQLModel = require('sequelize').Model;

module.exports = (sequelize, DataTypes) => {
  class Question extends SQLModel {
    /**
     * Helper method for defining associations.
     * This method is not a part of Sequelize lifecycle.
     * The `models/index` file will call this method automatically.
     */
  }
  Question.init(
    {
      question_id: {
        type: DataTypes.INTEGER,
        primaryKey: true
      },
      question_text: DataTypes.STRING,
    },
    {
      sequelize,
      tableName: 'final_question',
      modelName: 'Question',
      freezeTableName: true,
      timestamps: false,
    },
  );
  return Question;
};
